assets/messages.yml

```diff
@@ -1359,5 +1359,29 @@ messages:
 
         Please don't forget to search for an existing issue matching yours in the appropriate project before raising a new one.
 
+        %quick_links%
+      fillname: []
+  wrong-project-moved:
+    - project:
+        - bds
+        - mc
+        - mcd
+        - mce
```

Comment:
Minecraft Earth has been removed
```suggestion
```

Reply:
This is what I meant mostly, removing unnecessary projects (I said games by mistake lol)